akka-cluster-typed/src/main/resources/reference.conf

```diff
@@ -23,6 +23,8 @@ akka.cluster.typed.receptionist {
   # Settings for the Distributed Data replicator used by Receptionist.
   # Same layout as akka.cluster.distributed-data.
   distributed-data = ${akka.cluster.distributed-data}
+  # make sure that by default it's for all roles (Play loads config in different way)
+  distributed-data.role = ""
```

Comment:
Should it even be possible to change it from all nodes - override it in code instead?